casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala

```diff
@@ -23,15 +23,24 @@ import coop.rchain.crypto.{PrivateKey, PublicKey}
 import coop.rchain.crypto.codec.Base16
 import coop.rchain.crypto.signatures.Secp256k1
 import coop.rchain.rholang.interpreter.util.RevAddress
-import coop.rchain.shared.Log
-import monix.execution.Scheduler.Implicits.global
+import coop.rchain.shared.{Log, UncaughtExceptionLogger}
+import monix.execution.ExecutionModel.BatchedExecution
+import monix.execution.Scheduler
 import org.scalatest.{FlatSpec, Matchers}
 
 import scala.io.Source
 import scala.util.{Failure, Success, Try}
 
 class RholangBuildTest extends FlatSpec with Matchers {
 
+  val availableProcessors = java.lang.Runtime.getRuntime.availableProcessors()
+  implicit val scheduler = Scheduler.forkJoin(
```

Comment:
Why not move the one from `NodeRuntime` to some shared package and re-use here?

Reply:
PTAL